request/request.go

```diff
@@ -128,3 +128,13 @@ func GetExtraResources(req *v1beta1.RunFunctionRequest) (map[string][]resource.E
 	}
 	return out, nil
 }
+
+func GetCredentials(req *v1beta1.RunFunctionRequest) (map[string][]map[string][]byte, error) {
```

Comment:
```suggestion
func GetCredentials(req *v1beta1.RunFunctionRequest) (map[string][]resource.Credentials, error) {
```

I think we use this type from crossplane-runtime's pkg/resource elsewhere. It's just a `map[string][]byte` too but it at least makes this function signature look a little less intense. 🙂